app/src/main/res/drawable/login_background.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<inset xmlns:android="http://schemas.android.com/apk/res/android">
+    <shape
+        xmlns:android="http://schemas.android.com/apk/res/android"
+        android:shape="oval">
+
+    <solid
+        android:color="@android:color/white"/>
+
+    <size
+        android:width="30dp"
+        android:height="64dp"/>
+    </shape>
+</inset>
```

Comment:
Not any more. Was trying to draw the curve background in the mock but not able to get it working so gave up and used straight line instead..